### DISCUSSION TITLE
Add individual heading weight to MantineProvider

### DISCUSSION THREAD

**Discussion by StevenStavrakis** [mantinedev/mantine#1240] [Feature requests]
Currently, MantineProvider only tracks `fontSize` and `lineHeight` for headings. Designers often use weight to distinguish between headings, and control over the individual weight from the theme object should be considered a basic feature. While this can be handled in global CSS, I can't find a good reason why it should be. Putting heading weight controls in the theme would decrease the splitting of styles into multiple locations and allow for better control of type.

**Comment by rtivital** [comment]
We will consider adding this feature in one of 4.x releases.
